tests/file-match-crossed/test.yaml

```diff
@@ -16,26 +16,32 @@ checks:
         event_type: alert
         alert.signature_id: 2
   - filter:
+      # Suricata 7 does not manage to decode the second file (its size is 0)
```

Comment:
I think this other bug got fixed by moving mime parsing to rust in 8...

Reply:
do we need to fix that in 7?

Reply:
https://redmine.openinfosecfoundation.org/issues/7582